sda-commons-dependencies/build.gradle

```diff
@@ -9,50 +9,70 @@ javaPlatform {
 ext {
   awaitalityVersion = '4.2.0'
   logbackContribVersion = '0.1.5'
-  springBootVersion = '2.7.14'
-  springCloudVersion = '2021.0.8'
+  springBootVersion = '3.1.2'
```

Comment:
You can update to [3.1.3](https://central.sonatype.com/artifact/org.springframework.boot/spring-boot-dependencies/3.1.3/versions)